app-creator/src/widgets/ui-select/ui-select.ts

```diff
@@ -47,17 +52,21 @@ export class Select extends LitElement {
   onChangeHandler: (selection: string) => void = () => {};
 
   render() {
+    const { styles } = this;
     return html`
       <paper-dropdown-menu
         label="${this.placeholder}"
         ?disabled=${this.disabled}
         @value-changed=${this.handleSelectionChange}
+        style=${styleMap(styles)}
       >
         <paper-listbox
           slot="dropdown-content"
           selected="${this.items.indexOf(this.value)}"
         >
-          ${this.items.map((i) => html`<paper-item>${i}</paper-item>`)}
+          ${this.items
+            .split(', ')
```

Comment:
If I understand this correctly, users would have to type "item1, item2, ..." to make this work. Can we instead split **only** by comma without the whitespace, and then `trim()` the values to eliminate the whitespaces?

Reply:
+1. Side note: it also implies that a user can't have commas in their select, but this is fine for the first pass!